core/state_transition.go

```diff
@@ -378,7 +386,8 @@ func (st *StateTransition) refundGas(subnetEVM bool) {
 	}
 	// Return ETH for remaining gas, exchanged at the original rate.
 	remaining := new(big.Int).Mul(new(big.Int).SetUint64(st.gas), st.gasPrice)
-	st.state.AddBalance(st.msg.From(), remaining)
+	// This is fine against overflow since we already subtracted the fee from the sender
```

Comment:
this occurs at the end of the transaction execution, so I don't think this error is necessarily safe to ignore.

Reply:
this also changed